tests/components/home_connect/test_init.py

```diff
@@ -294,3 +306,66 @@ async def test_services_exception(
 
     with pytest.raises(ValueError):
         await hass.services.async_call(**service_call)
+
+
+TEST_UNIQUE_ID_PREFIX = "unique_id_prefix_"
+
+
+async def test_entity_migration(
+    hass: HomeAssistant,
+    device_registry: dr.DeviceRegistry,
+    entity_registry: er.EntityRegistry,
+    config_entry: MockConfigEntry,
```

Comment:
I think we should set the minor version of the config entry fixture to 2 and that we should create a new fixture here with version 1 and also assert that it got upgraded to v2